internal/sysfs/adapter_test.go

```diff
@@ -105,9 +107,7 @@ func TestAdapt_Open_Read(t *testing.T) {
 
 	// We can't correct operating system portability issues with os.DirFS on
 	// windows. Use syscall.DirFS instead!
-	if runtime.GOOS != "windows" {
-		testOpen_Read(t, testFS, true)
-	}
+	testOpen_Read(t, testFS, true, runtime.GOOS != "windows")
```

Comment:
notice we now make sure we do return inode from stat on windows. This saves us from doing a useless wasi-libc stat fan-out, except in the case of virtual files.